api/users/controller.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable func-names */
+/* eslint-disable no-confusing-arrow */
+const User = require('./model');
+
+exports.userCreate = function (req, res, next) {
+  const user = new User({
+    email: req.body.email,
+    firstName: req.body.firstName,
+    lastName: req.body.lastName,
+    nickNames: req.body.nickNames,
+    password: req.body.password,
+  });
+  user.save((err) => {
+    if (err) { return next(err); }
+    res.send({ message: 'New user created.' });
+  });
+};
+
+exports.userUpdate = function (req, res, next) {
```

Comment:
I haven't implemented an update yet but it will be a similar structure to `POST user`